src/scenarios/shared/postcommands.py

```diff
@@ -0,0 +1,7 @@
+from shared.const import APPDIR, TMPDIR, TRACEDIR, PUBDIR, BINDIR
+from performance.common import remove_directory
+
+def clean_directories():
```

Comment:
Thought: Is there a scenario where we do not want this to run in someone's `post.py`? If we always want it to run, we could leave this as the top level content of the script instead of inside a function, and then `post.py` is just `import shared.postcommands`.

Reply:
some very tentative thoughts...  1) keeping this means it could be expanded to add other functions like terminating servers, disconnecting the database...? (purely guess)
2) if this could be true, then maybe it's possible to just selectively do certain things in ```post.py``` and not deleting the folders 3) what's the advantage of having a light ```import shared.postcommands``` except it's only one line